pkg/schema/load/generic.go

```diff
@@ -144,7 +145,7 @@ func (gvs *genericVersionedSchema) TrimDefaults(r schema.Resource) (schema.Resou
 	if err != nil {
 		return r, err
 	}
-	rv, _, err := removeDefaultHelper(gvs.CUE(), rvInstance.Value())
+	rv, _, _, err := removeDefaultHelper(gvs.CUE(), rvInstance.Value())
```

Comment:
Nit. Following my other comment on return values. Why are the two booleans returned not important here.